validator/engine/htmlparser.js

```diff
@@ -898,9 +898,9 @@ amp.htmlparser.HtmlParser.INSIDE_TAG_TOKEN_ = new RegExp(
         ('(?:' +
          // Allow attribute names to start with /, avoiding assigning the / in
          // close-tag syntax */>.
-         '([^\\t\\r\\n /=>][^\\t\\r\\n =>]*|' +  // e.g. "href"
-         '[^\\t\\r\\n =>]+[^ >]|' +              // e.g. "/asdfs/asd"
-         '\/+(?!>))' +                           // e.g. "/"
+         '([^\\t\\r\\n /=>][^\\t\\r\\n =>]*|' + // e.g. "href"
+         '[^\\t\\r\\n =>]+[^ >]|' + // e.g. "/asdfs/asd"
+         '\/+(?!>))' + // e.g. "/"
```

Comment:
boo. sad lint can't accept this

Reply:
Fixed by turning off the `no-multi-spaces` rule and reverting this (and other similar) changes.